recipes/simple-build-dgcode/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "0.0.12" %}
+
+#NB get sha256 with: curl -sL <URL> | openssl sha256
+
+package:
+  name: simple-build-dgcode
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/simple-build-dgcode/simple-build-dgcode-{{ version }}.tar.gz
+  sha256: fdde85d7b07dda6b11e6b28bcdd2510c45afdd2d976a096640cf86f63adc0a75
+
+build:
+  skip: True  # [win or py<39]
```

Comment:
@jaimergp I will try, but I am not sure it will work since the simple-build-system package (a dependency) is not available on windows.

Reply:
@jaimergp Ok, great, that would be wonderful. I think I abandoned previous efforts since the window build failed and then I did not get all tests passed. But do you not think that this will cause problems in the future when I want to have to bots automatic update the versions (i.e. in those automatic bot PRs, will there not also be a spurious test failure for the windows builds?).

Reply:
(I added the same question in the main conversation thread of this PR, with a few more details)